subsys/net/lib/openthread/platform/uart.c

```diff
@@ -197,23 +193,66 @@ otError otPlatUartEnable(void)
 
 otError otPlatUartDisable(void)
 {
-	/* TODO: uninit UART */
-	LOG_WRN("%s not implemented.", __func__);
-	return OT_ERROR_NOT_IMPLEMENTED;
+#ifdef CONFIG_OPENTHREAD_NCP_SPINEL_ON_UART_ACM
+	int ret = usb_disable();
+
+	if (ret) {
+		LOG_WRN("Failed to disable USB, ret code %d", ret);
```

Comment:
BTW, for these kind of prints, I would recommend a shorter print (saves couple of bytes) as the "ret code %d" does not give much extra info. So something like this `LOG_WRN("Failed to disable USB (%d)", ret);` is usually enough.

Reply:
Done